gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -123,14 +131,7 @@ registration renew \
 
 The command creates a new registration file: `/home/user/flex/new/registration.yaml`.
 
-[NOTE]
-====
-Unsuccessful renewal attempts generate an empty `registration.yaml` file in the current path. Running `renew` again with this empty file in the current path results in the following error:
-
-`Error: file already exists: /renew/registration.yaml`
-
-Delete the empty `registration.yaml` file.
-====
+NOTE: In cases where the location of the created registration file differs from the runtime registration file location, you must copy the newly created file to the runtime registration file location.
```

Comment:
I'd try to make it simpler:
"NOTE: If the location of the created registration file differs from the runtime registration file location, you must copy the newly created file to the runtime registration file location."

Reply:
Good catch! Fixed.